lib/puppet/type/firewalld_service.rb

```diff
@@ -1,19 +1,20 @@
 require 'puppet'
 
 Puppet::Type.newtype(:firewalld_service) do
-  @doc = "Assigns a service to a specific firewalld zone.
-    firewalld_service will autorequire the firewalld_zone specified in the zone parameter and the firewalld::custom_service
-		specified in the service parameter, so there is no need to add dependencies for this
+  @doc = <<-EOM
+      Assigns a service to a specific firewalld zone.
+      firewalld_service will autorequire the firewalld_zone specified in the
+      zone parameter and the firewalld::custom_service specified in the service
+      parameter, so there is no need to add dependencies for this
 
-    Example:
+      Example:
```

Comment:
We can actually use puppet-strings tags instead.  Time to put my money where my mouth is though... So... https://github.com/voxpupuli/puppet-firewalld/pull/267